app/grandchallenge/reader_studies/models.py

```diff
@@ -828,34 +832,13 @@ class Meta:
 
     @cached_property
     def value_list(self):
-        cache_key = (
-            f"{self._meta.app_label}.{self._meta.model_name}-{self.pk}-"
-            f"{self.reader_study.modified.timestamp()}-"
-            f"{self.modified.timestamp()}"
-        )
-        cached = cache.get(cache_key)
-        if cached:
-            return cached
-        for key in cache.keys(
-            f"{self._meta.app_label}.{self._meta.model_name}-{self.pk}-*"
-        ):
-            cache.delete(key)
-        if self.is_editable:
-            items = self.reader_study.values_for_interfaces
-            for civ in self.values.all():
-                items[civ.interface.slug]["selected"] = civ.id
-                items[civ.interface.slug]["selected_image"] = (
-                    civ.image_id or ""
-                )
-        else:
-            items = {
-                val.interface.slug: {
-                    "title": val.title,
-                    "selected_image": val.image_id,
-                }
-                for val in self.values.all()
+        items = {
```

Comment:
This is much cleaner, the property name `value_list` is a little unclear as it returns a dict. I wonder if this is needed at all now? Could you go straight to iterating over the items? Where does `image_id` come from?

Reply:
I think this can probably be factored out. I'll have a look. `image_id` is provided by django for the foreignkey relation between civs ans images.